### DISCUSSION TITLE
Revert a page back to DNN

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1877] [Q&A] [answered]
I have a page that worked in DNN but not in Vanjaro. This is because it has the SEO module from 40Fingers on it which relies on a bunch of scripts that are not in Vanjaro.
So...is there a way to revert a Vanjaro page back to a DNN page?

**Comment by meetmandeep** [comment, marked as answer]
> is there a way to revert a Vanjaro page back to a DNN page?

There is no official way to revert back. You could update the Tabs table and set the SkinSrc to "NULL" for that specific TabID

**Reply by Tychodewaard** [reply]
Works. You have to clear some cache but after that, it's done :-)
